bundle/edu.gemini.spModel.io/src/main/java/edu/gemini/spModel/io/impl/migration/to2015B/WSCon.java

```diff
@@ -0,0 +1,225 @@
+package edu.gemini.spModel.io.impl.migration.to2015B;
+
+import java.awt.geom.*;
+
+// taken from wscon.java from NASA
+public class WSCon {
```

Comment:
FYI this code is copy/pasted from `wscon.java` and has a bunch of inlined methods to remove external dependencies. The testcase compares its output with that from Simbad and `jprecess` (which is apparently the standard implementation of this algorithm).